tests/utilities/test_apply_func.py

```diff
@@ -32,6 +45,12 @@ def test_recursive_application_to_collection():
         'e': np.array([10.]),  # numpy array
         'f': 'this_is_a_dummy_str',  # string
         'g': 12.,  # number
+        'h': Feature(input_ids=torch.tensor([1., 2., 3.]), segment_ids=np.array([4., 5., 6.])),  # dataclass
+        "i": ModelExample(
```

Comment:
could you standardize the string quotation marks here and below, sorry for the useless comment, our auto formatter does not yet do that automatically ^^